Gruntfile.js

```diff
@@ -22,10 +21,14 @@ module.exports = function(grunt){
   require('load-grunt-tasks')(grunt);
 
   grunt.registerTask('default', ['test-local', 'build']);
-  grunt.registerTask('sauce', ['karma:sauce-linux', 'karma:sauce-osx', 'karma:sauce-windows']);
-  grunt.registerTask('test-local', ['eslint', 'mochaTest:all', 'karma:local']);
-  grunt.registerTask('test-local-silent', ['eslint', 'mochaTest:silent', 'karma:local']);
-  grunt.registerTask('test', ['eslint', 'mochaTest:all']);
+  grunt.registerTask('sauce', [
+    'karma:sauce-linux',
+    'karma:sauce-osx',
+    'karma:sauce-windows'
```

Comment:
I'd encourage to add the dangling comma, as it's a good practice when using source control (adding it won't show a modified line every time we add a line to an array/object).

But this is something should be discussed, I guess (that's why I'm not setting this as a review comment).